McCorrections/include/TauTriggerSFs2017.h

```diff
@@ -63,9 +64,9 @@ class TauTriggerSFs2017
 
         // FIXME: Use the eta-phi efficiency corrections from pre-re-miniaod branch
         // Only medium, tight, and vtight are provided and they are from MVA ID
-//        std::string tauMVAWP = tauMVAWP_.data();
-//        if (tauMVAWP == "vvloose" || tauMVAWP == "vloose" || tauMVAWP == "loose") tauMVAWP = "medium";
-//        if (tauMVAWP == "vvtight")  tauMVAWP = "vtight";
+        std::string tau_MVAWP = tauMVAWP_.data();
+        if (tau_MVAWP == "vvloose" || tau_MVAWP == "vloose" || tau_MVAWP == "loose") tau_MVAWP = "medium";
+        if (tau_MVAWP == "vvtight")  tau_MVAWP = "vtight";
 
         // load the TH2s containing the eta phi efficiency corrections
         diTauEtaPhiData_ = loadTH2(inputFileOld_, Form("diTau_%s_DATA", tauMVAWP_.data()));
```

Comment:
Here in all lines below the new variable is used in the original code. Please cross-check.